### DISCUSSION TITLE
Can't enter REPL mode in VScode

### DISCUSSION THREAD

**Discussion by FlydGitHub** [micropython/micropython#11303] [RP2040 / Pico]
In Vstudio for Raspberry Pi RP2040 pico, there are two run buttons, the lover left hand side button connects, runs RP2040 - Micropython Code without error.

The top right "main" run button gives the error: ModuleNotFoundError: No module named 'machine' .

Any suggestions welcome.

Thanks

**Comment by bixb922** [comment]
If there is no module named machine, then probably you are connected to Python on the PC instead of Micropython on the RP2040. Have you tried with mpremote? Do you get REPL with mpremote?

**Comment by FlydGitHub** [comment]
It connects just fine the RP2040 using the lover button. Nether button does't not give me REPL.

I think it's a JSON config file issue.

How would REPL with mpremote work?
